src/stores/base.js

```diff
@@ -37,9 +37,9 @@ const useBaseStore = defineStore('base', {
 			logLevel: 'debug',
 			rf: {
 				region: undefined,
-				maxLongRangePowerlevel: undefined,
+				maxLongRangePowerlevel: 'auto',
 				txPower: {
-					powerlevel: undefined,
+					powerlevel: 'auto',
 					measured0dBm: undefined,
 				},
 			},
```

Comment:
@robertsLando is this enough to ensure users who haven't set this before now get auto-powerlevel enabled?

Reply:
Unfortunately nope: https://github.com/zwave-js/zwave-js-ui/blob/996e81a12666a7df45034009fafec029a065ad9e/src/stores/base.js#L508

Reply:
You could set it to default in `initSettings` by checking if it is undefined?

Reply:
The alternative is to use a merge module instead of object assign for this